README.md

```diff
@@ -24,281 +33,98 @@ of applications & services when it comes to rate limiting, authentication, autho
-kuadrant-system   ap-default-toystore   true
-```
-
-9.- Verify authentication
-
-Should return `401 Unauthorized`
+### Pre-requisites
+
+* Istio is installed in the cluster. Otherwise, refer to the
+[Istio getting started guide](https://istio.io/latest/docs/setup/getting-started/).
+* Kubernetes Gateway API is installed in the cluster. Otherwise,
+[configure Istio to expose a service using the Kubernetes Gateway API](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/).
+* Kuadrant is installed in the cluster.
+Otherwise, refer to the [kuadrant operator](https://github.com/Kuadrant/kuadrant-operator)
+for installation.
 
-```
-curl -v -H 'Host: api.toystore.com' -X POST http://localhost:9080/admin/toy
-```
+### If you are an *API Provider*
 
-Should return `200 OK` for alice
-
-```
-curl -v -H 'Host: api.toystore.com' -H 'Authorization: APIKEY ALICEKEYFORDEMO' -X POST http://localhost:9080/admin/toy
-```
+* Deploy the service/API to be protected ("Upstream")
+* Expose the service/API using the kubernetes Gateway API, ie
+[HTTPRoute](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.HTTPRoute) object.
+* Write and apply the Kuadrant's RateLimitPolicy custom resource targeting the HTTPRoute resource.
 
-Should return `200 OK` for bob
+Your service is being rate limited according to the policy you defined.
 
-```
-curl -v -H 'Host: api.toystore.com' -H 'Authorization: APIKEY BOBKEYFORDEMO' -X POST http://localhost:9080/admin/toy
-```
+### If you are a *Cluster Operator*
 
-10. Verify authenticated ratelimit by doing `200 OK` requests 2-3 times.
+* (Optionally) deploy istio ingress gateway using the
+[Gateway](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.Gateway) resource.
+* Write and apply the Kuadrant's RateLimitPolicy custom resource targeting the Gateway resource.
 
-## Demos
+Your gateway is being rate limited according to the policy you defined.
 
-### [Updating the RateLimitPolicy `targetRef` attribute](/doc/demo-rlp-update-targetref.md)
+## User guide index
 
-This demo shows how the kuadrant's controller applies the rate limit policy to the new HTTPRoute
-object and cleans up rate limit configuration to the HTTPRoute object no longer referenced by the policy.
+The user guides section of the docs gathers several use-cases as well as the instructions to implement them using kuadrant.
 
-### [Authenticated rate limiting](/doc/demo-rlp-authenticated.md)
+* [Simple rate limiting for API owners](doc/user-guides/simple-rl-for-api-owners.md)
+* [Authenticated rate limiting for API owners](doc/user-guides/authenticated-rl-for-api-owners.md)
+* [Gateway rate limiting for cluster operators](doc/user-guides/gateway-rl-for-cluster-operators.md)
 
-This demo shows how to configure rate limiting after authentication stage and rate limit configuration
-is per API key basis.
+## [Kuadrant Rate Limiting](/doc/rate-limiting.md)
+
+## FAQ
```

Comment:
Should we omit this section until we have content to add?

Reply:
done